apps/condo/domains/organization/integrations/sbbol/sync/syncUser.js

```diff
@@ -67,46 +67,50 @@ const cleanEmailForAlreadyExistingUserWithGivenEmail = async ({ email, userIdToE
  * @param dvSenderFields
  * @return {Promise<{importId}|*>}
  */
-const syncUser = async ({ context, userInfo }) => {
-    const returnFields = 'id phone email name importId importRemoteSystem'
-    const importFields = {
-        type: 'staff',
-        importId: userInfo.importId,
-        importRemoteSystem: userInfo.importRemoteSystem,
+const syncUser = async ({ context: { context, keystone }, userInfo }) => {
+    const identityWhereStatement = {
+        user: { type: STAFF },
+        identityId: userInfo.importId,
+        identityType: SBBOL_IDP_TYPE,
     }
-    const userFields = {
-        type: 'staff',
+    const userWhereStatement = {
+        type: STAFF,
         phone: userInfo.phone,
     }
-    const existingUsers = await getItems({
-        ...context,
-        listKey: 'User',
-        where: {
-            OR: [
-                { AND: userFields },
-                { AND: importFields },
-            ],
-        },
-        returnFields,
+
+    // let's search users by UserExternalIdentity and phone
+
+    const importedUsers = (await UserExternalIdentity.getAll(context, identityWhereStatement))
```

Comment:
we should have a db constrin for unique importId + importType

Reply:
it already exists